schutzbot/deploy.sh

```diff
@@ -33,6 +33,35 @@ sudo dnf install -y osbuild-composer composer-cli
 sudo mkdir -p /etc/osbuild-composer
 sudo cp -a schutzbot/osbuild-composer.toml /etc/osbuild-composer/
 
+sudo mkdir -p /etc/osbuild-worker
+
+# if GCP credentials are defined in the ENV, add them to the worker's configuration
+GOOGLE_APPLICATION_CREDENTIALS="${GOOGLE_APPLICATION_CREDENTIALS:-}"
+if [[ -n "$GOOGLE_APPLICATION_CREDENTIALS" ]]; then
+     # The credentials file must be copied to a different location. Jenkins places
+     # it into /tmp and as a result, the worker would not see it due to using PrivateTmp=true.
+     GCP_CREDS_WORKER_PATH="/etc/osbuild-worker/gcp-credentials.json"
+     sudo cp "$GOOGLE_APPLICATION_CREDENTIALS" "$GCP_CREDS_WORKER_PATH"
+     echo -e "\n[gcp]\ncredentials = \"$GCP_CREDS_WORKER_PATH\"\n" | sudo tee -a /etc/osbuild-worker/osbuild-worker.toml
+fi
+
+# if Azure credentials are defined in the env, create the credentials file
+AZURE_CLIENT_ID="${AZURE_CLIENT_ID:-}"
```

Comment:
If anything the set +x should go up here, no? Because here it "leaks" anyway.

Reply:
No, it's fine - https://osbuildci.cloud.paas.psi.redhat.com/blue/organizations/jenkins/osbuild%2Fimage-builder/detail/PR-156/7/pipeline/25#step-66-log-191

Reply:
Yeah, I didn't want to expose any secrets. `set +x` should definitely be several lines higher - I had it right originally but I think I got it wrong after several iterations.

Reply:
So it can be removed completely, right? Because Jenkins masks the "leaks" anyway.

Reply:
it's removed now...